### PR TITLE
#105 ctors for ethprivatekey

### PR DESCRIPTION
**Pull request by rockfridrich**
Solves #105 
Adds two new ctors for the private key to create it from `StringScalar` and `string`

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#107](https://github.com/blockstoreapp/web3swift/issues/107) is now in scope, role is `REV`

**Comment by biboran** [comment]
@rockfridrich please briefly describe how you solved the problem. Please prefix issue number with a hashtag.

**Comment by rockfridrich** [comment]
@biboran done

File: Example/Tests/PrivateKey/EthPrivateKeyTests.swift [hunk 1]
```diff
@@ -23,5 +23,31 @@ final class EthPrivateKeyTests: XCTestCase {
...
+        expect{
+            try EthPrivateKey(
+                hex: SimpleString{
+                    "0x1636e10756e62baabddd4364010444205f1216bdb1644ff8f776f6e2982aa9f5"
+                }
+                ).address().value()
+        }.to(
+            equal(
+                Data(hex: "0xcD8aC90d9cc7e4c03430d58d2f3e87Dae70b807e")
+            )
```
Comment (biboran) [inline Example/Tests/PrivateKey/EthPrivateKeyTests.swift]: @rockfridrich Please add test description

**Comment by rockfridrich** [comment]
@biboran fixed

**Comment by biboran** [comment]
@rockfridrich could you please pad the lines as I outlined above

**Comment by 0crat** [comment]
The job [#107](https://github.com/blockstoreapp/web3swift/issues/107) is now out of scope

**Pull request merged**